source/MaterialXGenGlsl/Nodes/LightCompoundNodeGlsl.cpp

```diff
@@ -94,10 +94,10 @@ void LightCompoundNodeGlsl::emitFunctionDefinition(const ShaderNode& node, Shade
     // Emit function definitions for each context used by this compound node
     for (int id : node.getContextIDs())
     {
-        const GenContext* context = shadergen.getNodeContext(id);
+        const GenContext* context = shadergen.getContext(id);
```

Comment:
We already renamed SgNodeContext to GenContext, so removing the word "Node" from the context methods to make this consistent.